pkg/webhook/server.go

```diff
@@ -280,6 +289,27 @@ func (s *Server) Start(ctx context.Context) error {
 	return nil
 }
 
+// StartedChecker returns an healthz.Checker which is healthy after the
+// server has been started.
+func (s *Server) StartedChecker() healthz.Checker {
+	return func(req *http.Request) error {
+		s.mu.Lock()
+		defer s.mu.Unlock()
+
+		if !s.started {
+			return fmt.Errorf("webhook server has not been started yet")
+		}
+
+		conn, err := net.DialTimeout("tcp", net.JoinHostPort(s.Host, strconv.Itoa(s.Port)), 10*time.Second)
```

Comment:
10 seconds is just a wild guess. Not sure after which threshold we want to declare the webhook server not ready.